recipe/build.sh

```diff
@@ -1,11 +1,17 @@
 set -e
 
+export OCL_ICD_VENDORDIR="${PREFIX}/etc/OpenCL/vendors"
+
+# Make sure `conda-build` keeps this directory.
+mkdir -p "${OCL_ICD_VENDORDIR}"
+touch "${OCL_ICD_VENDORDIR}/.conda_keep"
```

Comment:
Me too. Raising an issue ( https://github.com/conda-forge/ocl-icd-feedstock/issues/5 ) about it for now.